projects/helper/chain/radixdlt.js

```diff
@@ -37,12 +40,86 @@ async function queryAddresses({ addresses = [], }) {
   return items
 }
 
+async function queryLiquidStakeUnitDetails(addresses = []) {
+  /**
```

Comment:
Very nice! Thanks.

I'd suggest putting it above the function definition so it's a proper [JSDoc](https://jsdoc.app/about-getting-started)?

Also, I suggest we explain that the caller is responsible for validating that they are LSU resources, else this call could result in empty or partial returns.

Reply:
yep. makes sense to me